other/Settings.php

```diff
@@ -169,6 +169,12 @@
  * @var array
  */
 $cache_memcached = '';
+/**
+ * This is only used for redis. Should be a string of 'server:port'
+ *
+ * @var string
+ */
+$cache_redis = '';
```

Comment:
Just a thought, would it be better to use a generic variable here for all additional caching needs or leave it like this with additional settings.php variables?
The only downside to doing that is it prevents the possibility of using multiple different caching mechanisms at the same time.

Reply:
I thought about that as well, even moving the Redis specific parameters to a regular $modSettings key.

Will do the Redis implementation on another PR to keep things simpler but will most likely just use a regular $modSettings var instead.

Reply:
Will use a normal $modSettings var for this and will implement it on another PR.